### DISCUSSION TITLE
broken srcset for img when baseurl not top dir

### DISCUSSION THREAD

**Discussion by kalikaneko** [thuliteio/doks#249] [Q&A] [answered]
I'm unsure if this is really a bug, or I'm just missing something in the config, please advise:

i'm using the handy img shortcode, but all the resized img paths in the srcset after the 1st one are missing the baseUrl prefix, so they end up giving a 404 (since my site is not at the root folder of the server).

if it's really a bug, I can try to propse a patch.

btw, thanks for an awesome work!

**Comment by h-enk** [comment, marked as answer]
And thank you for using Doks  — I suppose you're using the child theme.

It's a bug, fixed in the starter theme (see https://github.com/h-enk/doks/pull/238), but not yet released to the child theme.

As a (temporary) quick fix add the [img shortcode template](https://github.com/h-enk/doks/blob/master/layouts/shortcodes/img.html) to your `./layouts/shortcodes/` folder.

**Reply by kalikaneko** [reply]
I see, I had missed the issue. Thanks!
